src/OOVPADatabase/D3D8LTCG/3911.inl

```diff
@@ -905,7 +905,7 @@ OOVPA_END;
 // * D3DDevice_SetTexture
 // ******************************************************************
 //81C10000F8FF89 ...C20400
-OOVPA_NO_XREF(D3DDevice_SetTexture_4, 2024, 9)
+OOVPA_NO_XREF(D3DDevice_SetTexture_0, 2024, 9)
```

Comment:
Here's what I suggest to change.

```suggestion
OOVPA_NO_XREF(D3DDevice_SetTexture_4__LTCG_eax_pTexture, 2024, 9)
```

Since non-LTCG version is actually 2 "push", 2\*4. And this call only did one push, 1\*4, and move pTexture to eax register.